twake/backend/node/test/e2e/channels/direct-channels.spec.ts

```diff
@@ -12,7 +12,7 @@ import { ChannelUtils, get as getChannelUtils } from "./utils";
 import { DirectChannel } from "../../../src/services/channels/entities/direct-channel";
 import { ChannelSaveOptions } from "../../../src/services/channels/web/types";
 
-describe("The direct channels API", () => {
+describe.only("The direct channels API", () => {
```

Comment:
This will skip all other tests

Reply:
It seems I forgot this one, thanks. (but actually it still do every other test files, the "only" is only working inside a single file)